### ISSUE TITLE
Behavior request: Pressing Tab Key behavior

### ISSUE THREAD

**Issue by jairusm** [ng2-ui/ng2-auto-complete#90]
Hi, thanks for this wonderful module. While using this i just have some observation while playing with the keyboard. It could be nice that when pressing the tab key it will pick out the highlighted item (same as pressing the enter key), just like the normal behavior of the select element. Just to have a more natural flow for better usability. :)

**Comment by allenhwkim** [comment]
I don't know how you get this idea, but with normal select item tab does not pick the currently highlighted item, but only ENTER does. 

ref; https://developer.mozilla.org/en/docs/Web/HTML/Element/select

**Comment by kevinhoelscher** [comment]
I'd like to point out that I tested the behavior of pressing [Tab] on the select control in the Examples section of the Mozilla page you linked, and it actually *does* update the selection to the highlighted item.  I verified this in all four browsers to which I have easy access (Chrome, Firefox, IE, Edge).  The only difference is in whether the focus remains in the select, or passes out to the next item on the page.

I think that the requested behavior would enhance the usability of a typical form.  In my experience, frequent web users are conditioned to associate [Enter] with a submit action.  Indeed, in many cases, pressing [Enter] anywhere else on the form will submit the form.

In most controls, a user can typically start typing until the control reflects the desired value and then press [Tab] to continue on.  I would suggest adding this as a configurable option, if not default behavior.  I am open to writing the change myself and submitting a pull request.
